gcore/src/lib.rs

```diff
@@ -67,6 +67,7 @@ pub mod errors;
 pub mod exec;
 pub mod msg;
 pub mod prog;
+pub mod stack_buffer;
```

Comment:
How do you feel about this?
```suggestion
pub mod stack;
```